app/app.component.ts

```diff
@@ -0,0 +1,16 @@
+import { HttpClient } from '@angular/common/http';
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { RouterModule, RouterOutlet } from '@angular/router';
+import { RegisterComponent } from "./components/register/register.component";
+
+@Component({
+    selector: 'app-root',
+    standalone: true,
+    templateUrl: './app.component.html',
+    styleUrl: './app.component.css',
+    imports: [RouterOutlet, FormsModule, RouterModule, RegisterComponent]
+})
+export class AppComponent {
+  title = 'firstapp';

```

Comment:
change this to Library Management System

Reply:
okay na